src/app_engine/decline_page.py

```diff
@@ -51,13 +51,26 @@ def post(self, room_id):
       self.write_response(constants.RESPONSE_INVALID_CALLEE)
       return
 
+    # Retrieve caller record.
+    caller_record = GCMRecord.get_by_gcm_id(caller_gcm_id, False)
     # Notify caller and other callee endpoints about the call decline.
-    gcm_ids_to_notify = [caller_gcm_id]
+    gcm_messages = []
```

Comment:
It would be helpful to split the comment and put `notify caller about decline` here and add `notify callee endpoints about decline` below (before the `for` loop).